src/find/matchers/size.rs

```diff
@@ -0,0 +1,174 @@
+use std::error::Error;
+use std::io::{stderr, Write};
+use walkdir::DirEntry;
+
+use find::matchers::{ComparableValue, Matcher, MatcherIO};
+
+#[derive(Clone, Copy, Debug)]
+enum Unit {
+    Byte,
+    TwoByteWord,
+    Block,
+    KiloByte,
```

Comment:
I could probably answer this myself from looking at the rest of the code, but should it be "Kilo" or "Kibi"?